### ISSUE TITLE
apply_central_impulse and friends don't wake bodies up

### ISSUE THREAD

**Issue by jFransham** [sebcrozet/nphysics#76]
As in title, if you call `apply_central_impulse` (or `apply_angular_momentum`, etc.) on a sleeping body nothing will happen, which is inconsistent with what happens when elements bump into eachother. For my usecase I just have

```rust
if &ActivationState::Inactive == handle.activation_state() {
     handle.activate(1.);
}
```

Above my calls to these methods, but that's obviously not The Right Thing.

**Comment by Ralith** [comment]
This can be closed now thanks to #80, right?

**Comment by sebcrozet** [comment]
Yes!
